File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/Y. The last 2 digits/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/Y

using System;

namespace Codeforces.Assuit.Sheet1
{
    class ProblemY
    {
        static void Main(string[] args)
        {
            string[] input = Console.ReadLine().Split(" ");

            int a = int.Parse(input[0]);
            int b = int.Parse(input[1]);
            int c = int.Parse(input[2]);
            int d = int.Parse(input[3]);

            a %= 100;
            b %= 100;
            c %= 100;
            d %= 100;

            int result = a * b % 100 * c % 100 * d % 100;
            Console.WriteLine(result.ToString("D2"));
        }
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/Y. The last 2 digits/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/Y

#include <iostream>
#include <cstdio>
using namespace std;

int main() {
    int A, B, C, D;
    cin >> A >> B >> C >> D;
    
    A %= 100;
    B %= 100;
    C %= 100;
    D %= 100;
    
    int result = (((A * B) % 100 * C) % 100 * D) % 100;
    printf("%02d", result);
    
    return 0;
}
